modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-field-type/src/main/resources/META-INF/resources/Text/Text.es.js

```diff
@@ -41,6 +40,7 @@ const Text = ({
 	syncDelay,
 	value: initialValue,
 }) => {
+	const {editingLanguageId} = useFormState();
```

Comment:
There's no need to use `useFormState()`, since the information you are looking for is available though the property `locale`